instrumentation/rediscala-1.8/src/main/java/io/opentelemetry/auto/instrumentation/rediscala/RediscalaInstrumentation.java

```diff
@@ -49,7 +50,7 @@
 public final class RediscalaInstrumentation extends Instrumenter.Default {
 
   public RediscalaInstrumentation() {
-    super("rediscala", "redis");
+    super("rediscala", Constants.REDIS);
```

Comment:
this happens to match the constant, but is not really related to db_system
```suggestion
    super("rediscala", "redis");
```